server/schema/Resolvers.ts

```diff
@@ -13,41 +13,41 @@ const resolvers: IResolvers = {
          */
         users: (): Promise<User[] | Error> =>
             UserDB.find()
-                .then((users: User[]): User[] => users)
+                .then((users: any): User[] => users)
                 .catch((err: Error): Error => new Error(`DB query failed: ${err}`)),
```

Comment:
Is this the quick fix to escaping the ts errors? I'm ok with it for now, it would be could if it was better typed but save that for later.

Reply:
yep, we can use the mongoose document type later down the line.